src/queries/auth/authMutations.ts

```diff
@@ -1,9 +1,27 @@
 import { UseMutationOptions, useMutation } from 'react-query';
-import { signInUser } from '@/data/auth';
-import { SignInUserData, SignInUserResponse } from '@/types/auth';
+import { loginUser, signUpUser, logoutUser } from '../../data/auth';
+import {
+  SignInUserData,
+  SignInUserResponse,
+  SignUpUserResponse,
+  SignUpUserData,
+  LogoutUserResponse,
+} from '../../types/auth';
 
 export function useSignInUserMutation(
   options?: UseMutationOptions<SignInUserResponse, Error, SignInUserData>,
 ) {
-  return useMutation('sign-in-user-mutation', signInUser, options);
+  return useMutation('sign-in-user-mutation', loginUser, options);
+}
+
+export function useSignUnUserMutation(
```

Comment:
I didn't notice the typo during the first review
```suggestion
export function useSignUpUserMutation(
```